packages/components/src/playground/button.spec.tsx

```diff
@@ -1,28 +1,33 @@
+import { primary, secondary } from "./button.stories";
 import { axe } from "jest-axe";
+import { prepareStory } from "@chanzuckerberg/story-utils";
 import { render } from "@testing-library/react";
-import { variants } from "./button.stories";
 
 describe("<Button />", () => {
-  it("renders the component", () => {
-    const { container } = render(variants());
+  it("renders primary buttons", () => {
+    const { container } = render(prepareStory(primary));
     expect(container.firstChild).toMatchInlineSnapshot(`
-      <div>
-        <button
-          class="button__ButtonComponent-ak17f2-0 cBvwNx"
-        >
-          Hello Button
-        </button>
-        <button
-          class="button__ButtonComponent-ak17f2-0 VxUNz"
-        >
-          Secondary Button
-        </button>
-      </div>
+      <button
+        class="button__ButtonComponent-ak17f2-0 cBvwNx"
+      >
+        Primary Button
+      </button>
+    `);
+  });
+
+  it("renders secondary buttons", () => {
+    const { container } = render(prepareStory(secondary));
+    expect(container.firstChild).toMatchInlineSnapshot(`
+      <button
+        class="button__ButtonComponent-ak17f2-0 VxUNz"
+      >
+        Secondary Button
+      </button>
     `);
   });
 
   it("passes accessibility checks", async () => {
```

Comment:
The secondary button doesn't actually appear to pass axe checks. But somehow this test still passes when I change it to secondary, instead of primary 😬

Reply:
eek o.o is this where we need axe-storybook

Reply:
Yeah, I think it's not high enough contrast, but it's not the real button anyway 😄